src/aggregate/aggregate.h

```diff
@@ -82,6 +82,12 @@ typedef enum {
   // Compound values are returned serialized (RESP2 or HASH) or expanded (RESP3 w/JSON)
   QEXEC_FORMAT_DEFAULT = 0x100000,
 
+  // Set the score of the doc to an RLookupKey in the result
+  QEXEC_F_SEND_SCORES_AS_FIELD = 0x200000,
```

Comment:
cherry-picking the `FT.AGGREGATE ADDSCORES` feature?

Reply:
Not intentionally. We can leave the flag without using it as suggested